templates/sidebar.html

```diff
@@ -130,15 +131,21 @@
                         <li class="nav-item">
                             <h5>Name: <span class="lead"> {{ profile.get_full_name }}</span> </h5>
                         </li>
+                    {% endif %}
```

Comment:
You don't need to end one if statement before starting the other. Nested if statements are allowed in django templates.

Reply:
> You don't need to end one if statement before starting the other. Nested if statements are allowed in django templates.

I tried removing it but I am getting error.

Reply:
I'm not saying to remove the endif statement, but to place that endif statement after the designation (that is, enclose the designation also in the first if statement).

Reply:
Ohh okay